### PR TITLE
Add reference to IS NULL operator from "Working with null" section

### PR DESCRIPTION
**Pull request by hedengran**

### CODE REVIEW

**Comment by hedengran** [comment]
[IS NULL and IS NOT NULL should be mentioned in 'working with null' section](https://trello.com/c/aIRkpFu8/543-is-null-and-is-not-null-should-be-mentioned-in-working-with-null-section)

File: cypher/cypher-docs/src/docs/dev/syntax/working-with-null.asciidoc [hunk 1]
```diff
@@ -93,3 +94,7 @@ a[coalesce($lower,0)..coalesce($upper,size(a))]
 * Arithmetic expressions containing `null`: `1 + null`
 * Function calls where any arguments are `null`: `sin(null)`
 
+[[is-null-is-not-null]]
+== Using `IS NULL` and `IS NOT NULL`
+Testing any value against `null` with both the `=` and the `<>` operators always evaluates to `null`.
+Instead, the special `IS NULL`, or `IS NOT NULL`, equality operators should be used (see <<cypher-comparison>>).
```
Comment (arnefischereit) [inline cypher/cypher-docs/src/docs/dev/syntax/working-with-null.asciidoc]:
```suggestion
Instead, the special `IS NULL`, or `IS NOT NULL` equality operators should be used (see <<cypher-comparison>>).
```

File: cypher/cypher-docs/src/docs/dev/syntax/working-with-null.asciidoc [hunk 2]
```diff
@@ -93,3 +94,7 @@ a[coalesce($lower,0)..coalesce($upper,size(a))]
 * Arithmetic expressions containing `null`: `1 + null`
 * Function calls where any arguments are `null`: `sin(null)`
 
+[[is-null-is-not-null]]
+== Using `IS NULL` and `IS NOT NULL`
+Testing any value against `null` with both the `=` and the `<>` operators always evaluates to `null`.
```
Comment (arnefischereit) [inline cypher/cypher-docs/src/docs/dev/syntax/working-with-null.asciidoc]: This sounds to me like you were talking about applying both operators at once.
```suggestion
Testing any value against `null` with either the `=` or the `<>` operator always evaluates to `null`.
```

**Review by arnefischereit — CHANGES_REQUESTED** [review]
Tiny correction.

File: cypher/cypher-docs/src/docs/dev/syntax/working-with-null.asciidoc (hunk 1 above)
Comment (AlexicaWright) [reply cypher/cypher-docs/src/docs/dev/syntax/working-with-null.asciidoc]:
```suggestion
Therefore,  use the special equality operators `IS NULL` or `IS NOT NULL` instead (see <<cypher-comparison>>).
```

File: cypher/cypher-docs/src/docs/dev/syntax/working-with-null.asciidoc (hunk 2 above)
Comment (AlexicaWright) [reply cypher/cypher-docs/src/docs/dev/syntax/working-with-null.asciidoc]:
```suggestion
Testing any value against `null`, either with the `=` operator or with the `<>` operator, always evaluates to `null`.
```

**Review by AlexicaWright — COMMENTED** [review]
Some editorial comments only.

**Review by arnefischereit — APPROVED** [review]
I approve of the new version, as suggested by @AlexicaWright

**Pull request merged**